index.html

```diff
@@ -520,7 +520,7 @@
     
 
     <link rel="stylesheet" href="eme.css">
-  <link rel="canonical" href="https://www.w3.org/TR/encrypted-media/"><style>/*
+  <meta name="description" content="This proposal extends  [HTML] providing APIs to control playback of encrypted content."><link rel="canonical" href="https://www.w3.org/TR/encrypted-media/"><style>/*
```

Comment:
maybe "... providing APIs to allow playback of encrypted content."

Reply:
Sure, but that's not related to this change, so let's follow-up on that separately.  Thanks!